### PR TITLE
feat: Create GliffCard

### PR DESCRIPTION
**Pull request by ChrisBaidoo**
## Description

Create a generic Dialog component  to use for popovers across the repos. 

closes #(issue-number)
relates gliff-ai/style#112 

## Checklist:

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request). If you're unsure about any of these, don't hesitate to leave a comment on this pull request!

- [x] I have read the gliff.ai Contribution Guide.
- [x] I have requested to **pull a branch** and not from main.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have performed a self-review of my own code.
- [ ] I have assigned **3 or less** reviewers.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] New database changes have been committed.
- [ ] If appropriate, I have bumped any version numbers.

### CODE REVIEW

File: src/GliffCard.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  headerTypography: {
+    fontSize: "21px",
+    width: "240px",
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  },
+});
+
+export function GliffCard(props: Props): JSX.Element {
```
Comment (ChasNelson1990) [inline src/GliffCard.tsx:42]: Do we not just want to call this Card? And should the props not inherit the MUI Card props?

**Comment by cooper667** [comment]
Can you add an example pls 😁

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gliff-ai/style",
-  "version": "4.0.1",
+  "version": "4.0.0",
```
Comment (cooper667) [inline package.json:3]: I don't think this should ever decrease :D

File: src/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    position: "relative",
+  },
+  headerTypography: {
+    fontSize: "21px",
+    width: "240px",
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  },
+});
```
Comment (cooper667) [inline src/Card.tsx:38]: I don't think we should create new components with `use\makeStyles`

**Review by cooper667 — CHANGES_REQUESTED** [review]

File: src/GliffCard.tsx (hunk 1 above)
Comment (cooper667) [reply src/GliffCard.tsx]: Rather than Card. Dialog or Modal? It doesn't actually replace the material card component. it's kind of a new one?

Comment (ChasNelson1990) [reply src/GliffCard.tsx]: MUI has four popover things, right, what is it most close to. Or nothing at all? @joshuajsmith what do you think?

Comment (cooper667) [reply src/GliffCard.tsx]: In fact, this raises the question of why we don't use the material UI dialog, as they have one...https://mui.com/components/dialogs/#customization

File: src/Misc.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  Chip,
+  Autocomplete,
+  Avatar,
+  Dialog,
+  InputLabel,
+  Select,
+  Alert,
+  Button,
+  MuiIconbutton,
+};
```
Comment (cooper667) [inline src/Misc.tsx:45]: I think these can just go straight into index.ts and use

`export {} from "@mul/material"` syntax

File: src/IconButton.tsx [hunk 5]
```diff
@@ -28,10 +28,11 @@ export interface Tooltip {
 export interface Props extends ButtonProps {
   component?: any;
   to?: string;
-  tooltip: Tooltip;
+  tooltip?: Tooltip;
```
Comment (cooper667) [inline src/IconButton.tsx:31]: do we support IconButtons with no tooltip?

File: src/Dialog.tsx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+const useStyle = makeStyles(() => ({
+  popoverPaper: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: "transparent",
+    borderRadius: "9px",
+  },
+}));
```
Comment (cooper667) [inline src/Dialog.tsx:32]: No `makeStyles` 😀

**Review by cooper667 — CHANGES_REQUESTED** [review]

**Review by cooper667 — APPROVED** [review]

**Review by SilviaZeta — APPROVED** [review]

**Pull request merged**